addons/mail/static/src/models/thread_view/thread_viewer.js

```diff
@@ -107,6 +107,13 @@ function factory(dependencies) {
         hasTopbar: attr({
             default: false,
         }),
+        /**
+         * States the sorting direction.
+         * Possible value: asc, desc
+         */
+        order: attr({
+            required: true,
+        }),
```

Comment:
make `asc` the default value and avoid repeating it on every test on which it doesn't actually matter

Reply:
This will fix runbot too because the issue is missing field